design model/components.model.json

```diff
@@ -194,18 +194,9 @@
                     "defaultValue": ""
                 },
                 {
-                    "name": "Months",
-                    "valueType": "number",
-                    "defaultValue": ""
-                },
-                {
-                    "name": "Layout",
-                    "valueType": "enum",
-                    "enumType": {
-                        "vertical": "Vertical",
-                        "horizontal": "Horizontal"
-                    },
-                    "defaultValue": "Horizontal"
+                    "name": "Vertical",
```

Comment:
This should be the consistent with the Calendar property. Does not have to match the Angular component property name.